src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -447,7 +456,7 @@ function getHermesCommand(): string {
     }
   };
   // assume if hermes-engine exists it should be used instead of hermesvm
-  const hermesEngine = path.join("node_modules", "hermes-engine", getHermesOSBin(), "hermes");
+  const hermesEngine = path.join("node_modules", "hermes-engine", getHermesOSBin(), `hermes${getHermesOSExe()}`);
```

Comment:
Let's have the method return the whole name, rather than just the extension. In general this change looks fine. We can release this quickly.